examples/text-sentiment/text_sentiment/runtime_model/hf_block.py

```diff
@@ -0,0 +1,92 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Standard
+import os
+
+# Third Party
+from text_sentiment.data_model.classification import (
```

Comment:
nit: this is local

Reply:
Good catch. Not sure if that was copy/paste or the formatter that did it! :)

Reply:
@evaline-ju This is `isort` changing it to make it `Third Party`. Seems like a bug to me.

Reply:
There's an isort config file where you can add known local folders

Reply:
@hickeyma I think you just need to add it as a known local in [the isort config](https://github.com/caikit/caikit/blob/main/.isort.cfg#L10). I don't think `isort` is smart enough to do this automatically

Reply:
ok, thanks

Reply:
Fixed